### ISSUE TITLE
Removing a person ( recognized face) from the classifier

### ISSUE THREAD

**Issue by peshmerge** [AISangam/Facenet-Real-time-face-recognition-using-deep-learning-Tensorflow#1]
Hello, 

First of all, thank for sharing this code :) It's really great! 
I have a question. 
If I train with 6 people [**Akshay Kumar, Nawazuddin Siddiqui, Salman Khan ,Shahrukh Khan ,Sunil Shetty, Sunny Deol**], just like what you have in you repo. Then a classifier.pkl file will be generated with information about those people! 
My question is: 
Can I remove only the second person (Nawazuddin Siddiqui) from the classifier.pkl without retraining the whole classifier? And do you know how can I do that? 

Thanks! 

Kind regards,
Pehsmerge

**Comment by AISangam** [comment]
Yes you can do it. There are many ways to solve this. But most simple way
to complete your goal (Removing a person ( recognized face) from the
classifier) is customise your prediction code according to your use case
instead of making some change on pkl file.

**Comment by peshmerge** [comment]
Hi, Thanks for your quick reply! 
But I don't get it! can you elaborate more on that? maybe more explanation of some (pseudo)code?
My goal is to remove a person completely from my model so it can't be recognized anymore! 

I could do it by retraining the model (.pkl file) from again with removing that person. The problem with this approach is if you have already trained your model with 10k people and you want to remove the person number 8020 then you have to exclude him/her and retrain the model (.pkl) with other 9999! That's really insane and stupid approach, tbh! 

Thanks!

**Comment by peshmerge** [comment]
You can download those files from David Sandberg repo (facenet) 
https://github.com/davidsandberg/facenet/tree/master/src/align
